templates/cts.html

```diff
@@ -45,4 +45,10 @@
 </style>
 
 <textarea id=results></textarea>
-<script type=module src=/webgpu/runtime/wpt.js></script>
+<script type=module>
+  import { setGlslangPath } from '/webgpu/framework/glslang_path.js';
```

Comment:
Huh my comment disappeared.

Unfamiliar with <script type="module">. Does setGlslangPath become global? Or is this an example and a browser using it would either replace this block entirely, or add another one?

Reply:
Imports are like `const`, iirc they don't escape the <script>.

The browser's fork of cts.html would insert new code inside this block.

Reply:
Thanks for the clarification, makes sense.

Reply:
https://chromium-review.googlesource.com/c/chromium/src/+/2100018/1/third_party/blink/web_tests/wpt_internal/webgpu/cts.html